src/components/DescriptionCard/index.tsx

```diff
@@ -7,18 +7,18 @@ type DescriptionCardProps = {
 
 export function DescriptionCard({ items }: DescriptionCardProps) {
   return (
-    <div className="flex flex-col">
-      <table className="table text-sm">
+    <div className="flex flex-col max-w-full">
+      <table className="table-auto shadow-none text-sm w-full max-w-full">
         <tbody>
           {items.map((item, index) => (
             <tr
               className="border-slate-100 border-solid border-b border-t p-1"
               key={item.label ?? "" + index}
             >
-              <th className="text-gray-700 font-light  uppercase text-left overflow-auto bg-zinc-50 pl-2">
+              <th className="text-gray-700 font-light  uppercase text-left break-all bg-zinc-50 pl-2">
```

Comment:
We don't want to break the labels - if anything we want to group by prefixes (labels split with a `/` or `:`)